climada/hazard/base.py

```diff
@@ -2183,7 +2157,6 @@ def _read_att_mat(data, file_name, var_names, centroids, description):
                 file_name, data[var_names['var_name']['ev_name']])
         except KeyError:
             attrs["event_name"] = list(attrs["event_id"])
-        attrs["description"] = description
         try:
             comment = u_hdf5.get_string(data[var_names['var_name']['comment']])
             attrs["description"] += ' ' + comment
```

Comment:
What is this attribute description? Is this not part of the tag?

Reply:
👍 I'll remove it as well.